buildpacks/nodejs-npm-install/src/bin/npm_runtime_config.rs

```diff
@@ -0,0 +1,52 @@
+use commons as _;
+use fastrand::alphanumeric;
+use fun_run::{CmdError, CommandWithName, NamedOutput};
+use heroku_nodejs_utils as _;
+use indoc as _;
+use libcnb as _;
+#[cfg(test)]
+use libcnb_test as _;
+use serde as _;
+#[cfg(test)]
+use serde_json as _;
+use std::env::temp_dir;
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::process::Command;
+#[cfg(test)]
+use test_support as _;
+
+fn main() {
+    if let Err(e) = disable_update_notifier() {
+        println!("{e}");
+    }
+
+    if let Err(e) = set_temp_cache_dir() {
+        println!("{e}");
+    }
+}
+
+fn disable_update_notifier() -> Result<NamedOutput, CmdError> {
+    Command::new("npm")
+        .args(["config", "set", "update-notifier", "false"])
```

Comment:
Does npm support configuring these via env vars instead? If so, would having these as build+launch env vars set during the build be simpler? eg:
https://docs.npmjs.com/cli/v10/using-npm/config#environment-variables

Reply:
@edmorley you're probably right. i'll give that a try.

Reply:
switched to using env vars. it's much simpler now. not sure what i was thinking with that exec.d script 😅